README.md

```diff
@@ -8,16 +8,61 @@
 
 ## Installation
 
-Install it yourself as:
-
     $ gem install rsgem
 
 We highly suggest to not include `rsgem` in your Gemfile.
 `rsgem` is not a library, and should not affect the dependency tree of your application.
 
 ## Usage
 
-_TODO_
+```
+rsgem new NAME [CI_PROVIDER]
+```
+
+RSGem will solve the following tasks for you:
+
+1. Create a folder for your gem leveraging bundler's defaults. (You need bundler in your system)
+1. Add the following dependencies:
+    - [Rake](https://github.com/ruby/rake)
+    - [Reek](https://github.com/troessner/reek)
+    - [RSpec](https://github.com/rspec/rspec)
+    - [Rubocop](https://github.com/rubocop-hq/rubocop)
+    - [Simplecov](https://github.com/colszowka/simplecov)
+        - We use Simplecov at 0.17.1 because that's the latest compatible version with CodeClimate.
+1. Add configuration files for Reek and Rubocop with default Rootstrap's configuration.
+1. Add a rake task to run Rubocop and Reek by calling `rake code_analysis`.
```

Comment:
```suggestion
1. Add a rake task to run Rubocop and Reek by calling `bundle exec rails code_analysis`.
```

Reply:
This is no rails, but we can add the bundle exec part as a good practice maybe